coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -38,33 +38,35 @@
             <span v-if="panel === index">Hide Documents</span>
             <span v-else>View Documents</span>
           </div>
-        </template>
-        <ul class="list document-list">
-          <li class="list-item"
-            v-for="(document, index) in item.filingDocuments"
-            v-bind:key="index">
-            <v-btn class="list-item__btn" flat color="primary" @click="downloadDocument(document)"
-              :disabled="loadingDocument" :loading="loadingDocument">
-              <img class="list-item__icon" src="@/assets/images/icons/file-pdf-outline.svg" />
-              <div class="list-item__title">{{document.name}}</div>
+        </v-expansion-panel-header>
+        <v-expansion-panel-content>
```

Comment:
These expansion panels need to be styled so they look same/similar as before:
- no drop shadow
- View Documents text pushed to the right
- line spacing for title/subtitle

Reply:
Shadow issue is resolved . The items needs to be aligned